pkg/etcd/etcd_suite_test.go

```diff
@@ -0,0 +1,86 @@
+package etcd
+
+import (
+	"testing"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	logf "sigs.k8s.io/controller-runtime/pkg/log"
+	"sigs.k8s.io/controller-runtime/pkg/log/zap"
+)
+
+// var (
```

Comment:
Please remove commented code blocks. No one will keep it up to date, it's better to recreate or copy from another place when needed